### PR TITLE
Gui: Make "Dock window overlay" GUI texts more consistent

### PR DESCRIPTION
**Pull request by Roy-043**

### CODE REVIEW

File: src/Gui/CommandView.cpp [hunk 1]
```diff
@@ -3925,8 +3925,8 @@ StdCmdDockOverlayMouseTransparent::StdCmdDockOverlayMouseTransparent()
   :Command("Std_DockOverlayMouseTransparent")
 {
   sGroup        = "View";
-  sMenuText     = QT_TR_NOOP("Bypass mouse events in dock overlay");
-  sToolTipText  = QT_TR_NOOP("Bypass all mouse events in dock overlay");
+  sMenuText     = QT_TR_NOOP("Bypass mouse events in docked overlays");
```
Comment (luzpaz) [inline src/Gui/CommandView.cpp:3928]: plural 'overlays' ? How do you figure ?

**Comment by adrianinsaval** [comment]
nitpicking but I don't think 'window' is the right word for dock widgets, perhaps use 'panel' instead?

**Comment by Roy-043** [comment]
In the original version the words "window", "widget" and "panel" are used. AFAICT the first two words were used interchangeably, I have therefore replaced "widget" with "window". It would surpise me if all 3 words have the same meaning in this context.

**Comment by wwmayer** [comment]
In Qt the class is called QDockWidget but in its description it says [dock windows](https://doc.qt.io/qt-6/qdockwidget.html#details). So, "windows" doesn't look wrong to me.

**Comment by realthunder** [comment]
I am fine changing the terminology. It is end user facing after all. Technically, my understanding is that Qt uses `Window` to refer to operating system backed user interface element, while `Widget` is what Qt implemented, sort of drawn on `Window`, which can host one or more `widgets`.

**Comment by adrianinsaval** [comment]
I'm not opposed to either term, and I'm not even native speaker so I could just be plain wrong. But to me panel communicates better what this does.

**Pull request merged**